packages/application-components/src/components/modal-pages/custom-form-modal-page/custom-form-modal-page.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { MessageDescriptor } from 'react-intl';
+import buttonMessages from '../../../utils/button-messages';
+import ModalPage from '../internals/modal-page';
+import ModalPageHeader from '../internals/modal-page-header';
+import { ContentWrapper } from '../internals/modal-page.styles';
+import { Spacings } from '@commercetools-frontend/ui-kit';
+import {
+  ConfirmButton,
+  CancelButton,
+  DeleteButton,
+} from '../../../utils/default-form-buttons';
+
+type Label = string | MessageDescriptor;
+type Props = {
+  level?: number;
+  title: string;
+  isOpen: boolean;
+  onClose?: (event: React.SyntheticEvent) => void;
+  children: React.ReactNode;
+  zIndex?: number;
+  baseZIndex?: number;
+  getParentSelector?: () => HTMLElement;
+  shouldDelayOnClose?: boolean;
+  // TopBar Props
+  topBarCurrentPathLabel?: string;
+  topBarPreviousPathLabel?: Label;
+  // Header Props
+  subtitle?: string | React.ReactElement;
+  // Controls Props
+  controls: React.ReactNode[];
```

Comment:
Why is this an array? Can't it just be one Node or would that mean the consumer has to often wrap in a `Fragment`?